src/Components/StatusBar/StatusBarMenu/StatusBarMenu.js

```
@@ -0,0 +1,125 @@
+        const left = (position.left + (position.right - position.left) / 2) - (MENU_WIDTH / 2);
+        setBottom(position.height);
+        setLeft(left);
+    };
+
+    const toggleMenu = () => {
+        setPositionOfMenu();
+        setShowMenu(!showMenu);
+    };
+
+    const sendToWorker = (code, args) => {
+        if (cdlWorker && cdlWorker.current) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)